packages/mask/src/plugins/FileService/Worker/swarm.ts

```diff
@@ -0,0 +1,82 @@
+import { Attachment } from '@dimensiondev/common-protocols'
+import { encodeText } from '@dimensiondev/kit'
+import { Bee, CollectionEntry } from '@ethersphere/bee-js'
+import urlcat from 'urlcat'
+import { isEmpty, times, sample } from 'lodash-unified'
+import { landing } from '../constants'
+import type { ProviderAgent, LandingPageMetadata, AttachmentOptions } from '../types'
+import { makeFileKeySigned } from '../helpers'
+
+const POSTAGE_STAMP = '0000000000000000000000000000000000000000000000000000000000000000'
```

Comment:
```suggestion
const POSTAGE_STAMP = '0'.repeat(64)
```

see https://mdn.io/string-repeat